src/checkers/inference/InferrableAnnotatedTypeFactory.java

```diff
@@ -0,0 +1,33 @@
+package checkers.inference;
+
+import org.checkerframework.framework.type.treeannotator.TreeAnnotator;
+
+/**
+ * Interface for indicating that the current the annotatedTypeFactory wish to be
```

Comment:
And, please don't anthropomorphize an AnnotatedTypeFactory.  (They *hate* it when you do that!)  This indicates a wish of the calling code.